packages/infra/lib/api-stack.ts

```diff
@@ -983,6 +1000,62 @@ export class APIStack extends Stack {
     return documentDownloaderLambda;
   }
 
+  private setupFhirToMedicalRecordLambda(ownProps: {
+    lambdaLayers: lambda.ILayerVersion[];
+    vpc: ec2.IVpc;
+    convertDocLambdaName: string;
+    converterUrl: string;
+    dynamoDBSidechainKeysTable: dynamodb.Table | undefined;
+    bucketName: string | undefined;
+    envType: string;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
```

Comment:
nit: it'd be good to have this as a separated file, we should break up this file so we can refactor how we build the infra, this would minimizing making it more complex.

Reply:
Created a ticket so we can get this out quicker https://github.com/metriport/metriport-internal/issues/1164